diff --git a/ios/components/map-view/MLRNMapView.m b/ios/components/map-view/MLRNMapView.m
index 22212748b1f1ca2eb73b5a1688ddb848b5c8ab0c..a9c2f392a1b99174d070c66616c838c2b2def2c0 100644
--- a/ios/components/map-view/MLRNMapView.m
+++ b/ios/components/map-view/MLRNMapView.m
@@ -23,7 +23,8 @@ @implementation MLRNMapView {
 static double const M2PI = M_PI * 2;
 
 - (instancetype)initWithFrame:(CGRect)frame {
-  if (self = [super initWithFrame:frame]) {
+  NSString *emptyStyle = @"{\"version\":8,\"sources\":{},\"layers\":[]}";
+  if (self = [super initWithFrame:frame styleJSON:emptyStyle]) {
     self.delegate = self;
 
     _pendingInitialLayout = YES;
